lms/services/grouping/service.py

```diff
@@ -203,6 +203,31 @@ def get_groups(
 
         return self._to_groupings(user, groupings, course, self.plugin.group_type)
 
+    def get_known_groupings(self, user, course):
```

Comment:
What do you mean by "known" here or "retrieve from the LMS" below? 

That we won't go and query the LMS API  to get the for example sections of a course?

Reply:
Yes that's right. I've updated the comment to hopefully be clearer.